### ISSUE TITLE
Link to doc GraphiQL instance references the wrong host

### ISSUE THREAD

**Issue by skokenes** [stefanwalther/qix-graphql#34]
**Describe the bug**
When following along with the README, it shows how you can use
```
_links {
  _doc
}
```
to get a url to a GraphiQL instance connected to a specific doc. In the README, the example gets hosted locally at `localhost:3004`. However, the response url that the API provides references `qix-graphql` instead of `localhost` as the host. Ex:
```
{
  "data": {
    "doc": {
      "_links": {
        "_doc": "http://qix-graphql:3004/app/%2Fdocs%2FConsumer%20Goods%20Example.qvf/graphql"
      }
    }
  }
}
```

**To Reproduce**
1. Follow the instructions on the README to spin up GraphiQL
2. Run a query like so to get the link for a doc:
```
{
  doc(qDocId: "/docs/Consumer Goods Example.qvf") {
    _links {
      _doc
    }
  }
}
```
3. Look at the URL that is returned by the API call

**Expected behavior**
The URL should reference `localhost` as the host, not `qix-graphql`.

**Comment by stefanwalther** [comment]
Thx a lot, good catch, will fix ASAP.

**Comment by stefanwalther** [comment]
Should be fixed with #36 .
Please feel free to create a new issue or re-open if this is not the case.
